### PR TITLE
failed compilation on previous pull request, not sure why, submitting again

### PR DESCRIPTION
**Pull request by MazinKarjikar**

### CODE REVIEW

**Comment by MazinKarjikar** [comment]
I formatted a percent sign incorrectly, fixed it using correct latex format

File: codes/quantum/qubits/surface/surface/surface.yml [hunk 1]
```diff
@@ -134,6 +134,8 @@ realizations:
     One cycle of syndrome readout on 19-qubit planar and 24-qubit toric codes realized in two-dimensional Rydberg atomic arrays \cite{arxiv:2112.03923}.
   - |
     Signatures of corresponding topological phase of matter detected in superconducting circuits \cite{arxiv:2104.01180} and two-dimensional Rydberg atomic arrays \cite{arXiv:2104.04119}.
+  - |
```
Comment (valbert4) [inline codes/quantum/qubits/surface/surface/surface.yml:137]: This has already been added in code_id: stab_4_2_2. In the future, please do a repo-wide search of the arxiv reference before you add it.

File: codes/quantum/qubits/subsystem/bacon_shor.yml [hunk 2]
```diff
@@ -43,6 +43,7 @@ features:
 
 realizations:
   - 'Trapped-ion qubits: state preparation, logical measurement, and stabilizer measurement for nine-qubit Bacon-Shor code demonstrated on a 13-qubit device by M. Cetina and C. Monroe groups \cite{arxiv:2009.11482}.'
+  - 'Photons: quantum teleportation of information implemented on maximally entangled pair of physical and logical qubits with fidelity rate of up to 78.6\% \cite{arXiv:2009.06242}.'
```
Comment (valbert4) [inline codes/quantum/qubits/subsystem/bacon_shor.yml:46]: Nice. Please list how many physical qubits.

File: codes/quantum/qubits/quantum_repetition.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ protection: 'Bit-flip code detects bit-flip errors \(X\) on \(\left\lfloor (n-1)
...
-  - 'Superconducting circuits: 3-qubit phase-flip and bit-flip code by Schoelkopf group \cite{arXiv:1109.4948}, IBM 15-qubit device \cite{arXiv:1709.00990}, Google experiment performing up to 8 error-correction cycles on 5 and 9 qubits \cite{arXiv:1411.7403}, and Google Quantum AI Sycamore utilizing from 5 to 21 qubits \cite{arXiv:2102.06132}. Continuous error correction protocols have been implemented on a 3-qubit superconducting circuit device \cite{arxiv:2107.11398}.'
+  - 'Superconducting circuits: 3-qubit phase-flip and bit-flip code by Schoelkopf group \cite{arXiv:1109.4948}, another 3-qubit bit-flip code \cite{arXiv:1411.5542}, 3-qubit phase-flip code up to 3 cycles of error correction \cite{arXiv:1508.01388}, IBM 15-qubit device \cite{arXiv:1709.00990}, IBM 43-qubit code on Rochester device \cite{arXiv:2004.11037}, Google Quantum AI Sycamore performing up to 8 error-correction cycles on 5 and 9 qubits \cite{arXiv:2102.06132}, Google Quantum AI Sycamore utilizing from 5 to 21 qubits \cite{arXiv:2102.06132}. Continuous error correction protocols have been implemented on a 3-qubit superconducting circuit device \cite{arxiv:2107.11398}.'
```
Comment (valbert4) [inline codes/quantum/qubits/quantum_repetition.yml:31]: "IBM Rochester device"
Also, you've deleted 1411.7403, which is a separate ref from 2102.06132, which does much larger codes than 5 or 9 physical qubits. Please fix.

File: codes/quantum/qubits/color.yml [hunk 4]
```diff
@@ -47,7 +47,8 @@ features:
     - '\(0.143\%\) with depolarising circuit-level noise using perfect-matching decoder \cite{arXiv:1407.5103}.'
     - '\(>0\%\) threshold with sweep decoder \cite{doi:10.7907/059V-MG69}.'
 
-#realizations:
+realizations:
```
Comment (valbert4) [inline codes/quantum/qubits/color.yml:50]: This is already in stab_4_2_2

File: codes/quantum/properties/quantum_perfect.yml [hunk 5]
```diff
@@ -31,6 +31,7 @@ features:
 realizations:
  - 'NMR: Implementation of identity, NOT, and Haddamard gates along with perfect error correcting code on single qubit errors in 5 qubit circuit \cite{arXiv:1208.4797}.'
  - 'NMR: Implementation of perfect error correcting code on 5 spin subsystem of labeled crotonic acid for quantum network benchmarking \cite{arXiv:quant-ph/0101034}.'
+ - 'Superconducting circuits: Implementation of smallest perfect code on 5-qubit circuit with 74.5\% overall fidelity rate \cite{arXiv:1907.04507}.'
```
Comment (valbert4) [inline codes/quantum/properties/quantum_perfect.yml:34]: This is in stab_5_1_3

**Comment by valbert4** [comment]
Thanks. Please resubmit with changes.

File: codes/quantum/qubits/quantum_repetition.yml (hunk 3 above)
Comment (MazinKarjikar) [reply codes/quantum/qubits/quantum_repetition.yml:31]: Fixed this in the new commit, sorry about that.

File: codes/quantum/qubits/subsystem/bacon_shor.yml (hunk 2 above)
Comment (MazinKarjikar) [reply codes/quantum/qubits/subsystem/bacon_shor.yml:46]: It was one of each, I listed it in the recent commit.

**Pull request closed without merging**